contrib/verifybinaries/test.py

```diff
@@ -8,19 +8,24 @@
 
 def main():
     """Tests ordered roughly from faster to slower."""
+    if subprocess.run("gpg --list-keys | grep 0x90C8019E36C2E964 >/dev/null", shell=True).returncode != 0:
```

Comment:
On macOS gpg (GnuPG/MacGPG2) 2.2.40 has a slightly different output format for `--list-keys`. It won't include the `0x`. The manual also recommends adding `--with-colons` for a machine-readable format that's meant to stay consistent.

So maybe try: `gpg --list-keys --with-colons | grep 90C8019E36C2E964 >/dev/null`

Reply:
With that change, the 0.22.0 test fails for me as before.

Reply:
> So maybe try: `gpg --list-keys --with-colons | grep 90C8019E36C2E964 >/dev/null`

Tested and also working on Ubuntu

Reply:
Test also fails on a (relatively) fresh Ubuntu 22.10 machine. As does `contrib/verifybinaries/verify.py 0.20.0`.